homeassistant/components/light/__init__.py

```diff
@@ -107,6 +110,7 @@
     ATTR_WHITE_VALUE: vol.All(vol.Coerce(int), vol.Range(min=0, max=255)),
     ATTR_FLASH: vol.In([FLASH_SHORT, FLASH_LONG]),
     ATTR_EFFECT: cv.string,
+    vol.Optional(ATTR_POWER_ON, default=True): cv.boolean,
```

Comment:
btw, this idea came up before and I have shot it down because of proposed implementations (things like a set_state command etc).

As far as I know LIFX is the only light that allows changing settings while it's off. So I suggest you add it as a LIFX only service. If we ever have more lights than we can maybe generalize it.